packages/shared/src/enums.ts

```diff
@@ -53,6 +60,13 @@ export const assetConstants = {
     decimals: 6,
     contractId: 3,
   },
+  [InternalAssets.Usdt]: {
+    chain: Chains.Ethereum,
+    asset: Assets.USDT,
+    name: 'USDT',
+    decimals: 6,
+    contractId: 6, // TODO: update this once the final contract id is merged into the backend repo
```

Comment:
nee to update this once the assets can be found in the backend: https://github.com/chainflip-io/chainflip-backend/blob/4540356c923f30777aea46446b81066db1203a6d/state-chain/primitives/src/chains/assets.rs#L402